docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -134,18 +134,32 @@ testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 
 ### Starting browser with arguments
 
-If you need to pass arguments for the specified browser, write them right after browser alias. Surround the browser call and its arguments with quotation marks:
+If you need to pass arguments for the specified browser, write them right after browser alias. Surround the browser call and its arguments with quotation marks. On Windows, use double quotation marks.
 
 ```sh
 testcafe "chrome --start-fullscreen" tests/sample-fixture.js
 ```
 
+On `*nix` systems, prefer single quotation marks.
```

Comment:
In common Unix shells, like `bash`, `zsh`, `csh` (defaults on macOS, Linux, Windows Subsystem for Linux) and Windows PowerShell, prefer single quotation marks.